PINCE.py

```diff
@@ -1942,6 +1953,25 @@ def treeWidget_AddressTable_edit_value(self):
                 debugcore.write_memory(address, vt.value_index, parsed_value, vt.zero_terminate, vt.endian)
             self.update_address_table()
 
+    def treeWidget_AddressTable_edit_offset(self):
+        row = guiutils.get_current_item(self.treeWidget_AddressTable)
+        if not row:
+            return
+
+        dialog = InputDialogForm(self, item_list=[("Offset addresses by:",'')])
+        if dialog.exec():
+            offset_value = dialog.get_values()
+            offset_int = int(offset_value, 16)
+            print(offset_value)
```

Comment:
Remove debug prints once you're done with them.

Reply:
Removed